src/core/device.ts

```diff
@@ -105,6 +101,11 @@ export const ready = (() => {
   // must be extracted from global scope object as otherwise we get ReferenceError in node.js
   const document: Document = global['document'];
   const window: Window = global['window'];
+  const callback = () => {
+    document.removeEventListener('load', callback);
+    document.removeEventListener('DOMContentLoaded', callback);
+    return Q.resolve(document);
```

Comment:
This is not the same code, the original called the resolve function passed to the promise constructor. I'm surprised this works at runtime!?!